Description: Workaround for the automake1.11-2 bug
 This works around the automake bug in the automake1.11-2 which does not 
 recognize the prefix/primary combination `pkglibexec_SCRIPTS' as valid.
 This bug was fixed in the automake1.11-2b release.
Author: Aron Xu <happyaron.xu@gmail.com>, Daiki Ueno <ueno@unixuser.org>

--- a/ui/gtk/Makefile.am
+++ b/ui/gtk/Makefile.am
@@ -36,7 +36,8 @@
 	$(NULL)
 ui_gtkdir = $(pkgdatadir)/ui/gtk
 
-pkglibexec_SCRIPTS = \
+ibusuigtkdir = $(pkglibexecdir)
+dist_ibusuigtk_SCRIPTS = \
 	ibus-ui-gtk \
 	$(NULL)
 
